controller/tests/array_action/ds8k/test_array_mediator_ds8k.py

```diff
@@ -180,19 +210,26 @@ def test_create_volume_fail_with_pool_not_found(self):
             self.array.create_volume("fake_name", 1, 'thin', "fake_pool")
 
     def test_create_volume_fail_with_incorrect_id(self):
-        self.client_mock.get_volumes_by_pool.side_effect = InternalServerError("500", message="BE7A0005")
+        self.client_mock.create_volume.side_effect = InternalServerError("500", message="BE7A0005")
```

Comment:
I think the `get` would fail for an invalid id before we reach `create`

Reply:
that is a unit test.. in create_volume there is no get